pkg/log/log.go

```diff
@@ -0,0 +1,44 @@
+package log
```

Comment:
I think that this entire module is unnecessary for now. We won't build a logging abstraction layer so we should simply embrace logr. I would update the README to mention that we use logr internally and give a quick example of how one should call logr.NewContext to pass us a logger. GetLogger should be part of the example CLI and we should call logr.FromContext directly from the code. No need to allocate a logger and force logging on library users. We also shouldn't change the name. To expose the protocol anme, use logr.WithValues